hibernate-core/src/main/java/org/hibernate/type/descriptor/jdbc/internal/JdbcTypeBaseline.java

```diff
@@ -85,12 +85,12 @@ public static void prime(BaselineTarget target) {
 		target.addDescriptor( BinaryJdbcType.INSTANCE );
 		target.addDescriptor( VarbinaryJdbcType.INSTANCE );
 		target.addDescriptor( LongVarbinaryJdbcType.INSTANCE );
-		target.addDescriptor( new LongVarbinaryJdbcType(SqlTypes.LONG32VARBINARY) );
+		target.addDescriptor( new LongVarbinaryJdbcType(SqlTypes.LONGVARBINARY, SqlTypes.LONG32VARBINARY) );
```

Comment:
@cigaly this looks fine, but would a simpler (and maybe slightly better) change be to just register the type twice, something like:

```java
target.addDescriptor( new LongVarbinaryJdbcType(SqlTypes.SqlTypes.LONG32VARBINARY) );
target.addDescriptor( SqlTypes.LONG32VARBINARY, new LongVarbinaryJdbcType(SqlTypes.LONG32VARBINARY) );
```

(I'm not sure, just asking.)

Reply:
Oh, no, wait, scratch that, I misunderstood.